aikau/src/main/resources/alfresco/forms/Form.js

```diff
@@ -309,7 +310,27 @@ define(["dojo/_base/declare",
        * @since 1.0.32
        */
       warningsPosition: "top",
-      
+
+      /**
+       * Should the first field in the form be focused when the form is loaded?
+       *
+       * @instance
+       * @type {boolean}
+       * @default
+       * @since 1.0.40
+       */
+      firstFieldFocusOnLoad: false,
```

Comment:
We're now trying to ensure that variables are listed alphabetically rather than randomly (yes, this isn't applied throughout the code base entirely, but you'll see that with each PR we are making updates the code).